modules/base/rendering/renderablesphere.cpp

```diff
@@ -161,6 +177,12 @@ namespace {
         // [[codegen::verbatim(FadeOutThresholdInfo.description)]]
         std::optional<float> fadeOutThreshold [[codegen::inrange(0.0, 1.0)]];
 
+        // [[codegen::verbatim(TransferFunctionPathInfo.description)]]
+        std::optional<std::filesystem::path> colorMap;
+
+        // [[codegen::verbatim(IsUsingColorMapInfo.description)]]
+        std::optional<bool> useColorMap;
```

Comment:
See comment above, but this name must match the identifier of the property that the verbatim is used for, if I remember correctly
```suggestion
        // [[codegen::verbatim(UseColorMapInfo.description)]]
        std::optional<bool> useColorMap;
```